### ISSUE TITLE
FreeBSD: After server upgrade, login screen is split.

### ISSUE THREAD

**Issue by kulmosen** [Ylianst/MeshCentral#3795]
<img width="1492" alt="Skærmbillede 2022-03-24 kl  11 03 24" src="https://user-images.githubusercontent.com/10152058/159892481-3c973f05-bfa7-4a22-ad0e-4fc7cdebadf1.png">

After upgrading the FreeBSD host to 13.0-RELEASE, the Meshcentral login is split and every part is loaded in one window. Look at attached picture. 
Meshcentral is installed in JAIL, which was not upgraded in first place. After issue occurred, i decided to try and upgrade the JAIL along side with NODE to latest version. But that didnt help. 
All buttons are non responsive.

Issued a
``` 
npm update
npm install meshcentral
```
Which didn't solve the issue either.
Has anyone had the same experience?

**Comment by JSuenram** [comment]
Looks like the page is not rendered correctly.... maybe make a backup of DB and Config and reinstall MeshCentral from scratch and put config and DB back after.....

**Comment by Ylianst** [comment]
Looks like something is corrupt. First, check that you server disk storage is not full, that will cause MeshCentral updates to mess up like that. Next go in the folder above "node_modules" and do something like this:

```
systemctl stop meshcentral
mv node_modules node_modules_bak
npm install meshcentral
node node_modules/meshcentral
  (test to make sure everything works, hit ctrl-c when done)
systemctl start meshcentral
```

This should reset all of the MeshCentral files. As always, good idea to keep an offsite backup of the "meshcentral-data" folder even if this operation does not touch any of the data.

**Comment by kulmosen** [comment]
The server still has 59Gb available storage.
I did:
```
cd meshcentral
pm2 stop meshcentral
mv node_modules node_modules.bak
npm install meshcentral
node node_modules/meshcentral
```
But with no luck. 
The output of the last command is:
```
node node_modules/meshcentral
Installing archiver@4.0.2...
Installing otplib@10.2.3...
MeshCentral HTTP redirection server running on port 800.
MeshCentral v1.0.1, Hybrid (LAN + WAN) mode.
MeshCentral Intel(R) AMT server running on support.solutiogruppen.dk:4433.
MeshCentral HTTP server running on port 4430, alias port 443.
Loaded web certificate from "https://support.solutiogruppen.dk:443/", host: "support.solutiogruppen.dk"
  SHA384 cert hash: >> hash snipped out :-) <<
```
The Meshcentral is located behind a NGINX reverse proxy, with Let'sEncrypt certificates.

**Comment by kulmosen** [comment]
So finally found the issue at hand. If others is having the same issue, then the solution is documented.
After upgrading the meshcentral including node and npm, something broke when using my NGINX reverse proxy.
After a bit of troubleshooting, including connecting directly to the meshcentral server IP adress and port, I did find that it worked when bypassing the NGINX reverse proxy.
So I noticed that other had caching issues when having sendfile = on;
So i tried turning sendfile to off, but with no luck.

So digging through the NGINX reverse proxy config file for my meshcentral server, showed that I was missing some settings. So I added the following:

```
http {
        sendfile  off;
}
server {
        client_max_body_size 10G;
        underscores_in_headers on;
}

location / {
        add_header Front-End-Https on;
}
```
And after this, everything worked as a charme. 
I have no idea which of the settings added are the one and only or if its a combination of either one. In my case I am back in bussiness. :-)
